app/components/avo/fields/trix_field/show_component.html.erb

```diff
@@ -1,14 +1,13 @@
 <%= field_wrapper **field_wrapper_args do %>
-  <%
-    content_classes = 'trix-content py-2 max-w-4xl'
-    content_classes << ' hidden' unless @field.always_show
-  %>
-  <div data-controller="hidden-input">
-    <% unless @field.always_show %>
-      <%= link_to t('avo.show_content'), 'javascript:void(0);', class: 'font-bold inline-block', data: { action: 'click->hidden-input#showContent' } %>
-    <% end %>
-    <div class="<%= content_classes %> " data-hidden-input-target="content">
+  <div data-controller="trix-body">
+    <div class="trix-content py-2 max-w-4xl hidden" data-trix-body-target="content">
       <%= sanitize @field.value.to_s %>
     </div>
+    <div class="hidden" data-trix-body-target="moreContentButton">
+      <%= link_to t('avo.more_content'), 'javascript:void(0);', class: 'font-bold inline-block pt-3', data: { action: 'click->trix-body#toggleContent' } %>
+    </div>
+    <div class="hidden" data-trix-body-target="lessContentButton">
+      <%= link_to t('avo.less_content'), 'javascript:void(0);', class: 'font-bold inline-block pt-3', data: { action: 'click->trix-body#toggleContent' } %>
+    </div>
```

Comment:
`@field.always_show` seems to be ignored after these changes.

`always_show` is an option for the Trix field, it is documented here https://docs.avohq.io/3.0/fields/trix.html#always_show

We should ensure that setting `always_show` to true bypasses the entire show/hide content logic.